src/app/services/graph.service.js

```diff
@@ -174,21 +174,9 @@ angular
                     }
                 },
                 {
-                    selector: 'node[resource_type="source"]',
+                    selector: 'node[color]',
```

Comment:
this is a selector for any node with a `color` property defined. I tried to do `node[docs.color]`, but I don't think nested selectors are allowed!